app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationList.kt

```diff
@@ -71,6 +75,15 @@ fun ConversationList(
     onAudioPermissionPermanentlyDenied: () -> Unit = {}
 ) {
     val context = LocalContext.current
+    var listScrollState by remember { mutableStateOf(false to 0) }
```

Comment:
what is your take on the `Pair` ? I'd say its enough for our intermediate state here, but we can always change it to a dedicated data class if we need some more context